src/controllers/controller.cpp

```diff
@@ -83,6 +84,13 @@ bool Controller::applyMapping() {
     m_pScriptEngineLegacy->setScriptFiles(scriptFiles);
 
     m_pScriptEngineLegacy->setSettings(pMapping->getSettings());
+#ifdef MIXXX_USE_QML
+    m_pScriptEngineLegacy->setLibraryDirectories(pMapping->getLibraryDirectories());
```

Comment:
Can you help me understand why this line is here?  Why does the script engine need the library directories?

Reply:
Probably poor naming on my end - "Library Directory" refers to custom QML library directories defined in the mapping manifest. This allows developers to create QML modules and split/reuse component for a screen.

I've renamed it to "modules" (setModules/getModules) so hopefully it is less confusing